pkg/grpc/device_definition.proto

```diff
@@ -241,6 +241,31 @@ message GetDeviceStyleByIDRequest {
 	string id = 1;
 }
 
+message GetDeviceCompatibilityListRequest {
+	string make_id = 1;
+	string integration_id = 2;
+	string region = 3;
+}
+
+message Feature {
+	string key = 1;
+	int32 support_level = 2;
+}
+
+message DeviceModelYears {
```

Comment:
```suggestion
message DeviceModelYear {
```

Reply:
Well, apart from being plural or not maybe this ought to be more specific? Maybe "compatibility" ought to be in the name.